packages/editor-ui/src/components/Node.vue

```diff
@@ -214,6 +243,75 @@ export default mixins(externalHooks, nodeBase, nodeHelpers, workflowHelpers).ext
 				this.setSubtitle();
 			}
 		},
+		isDragActiveTriggerNode: {
+			handler(isDragActiveTriggerNode) {
+				if (this.workflowRunning && this.isTriggerNode && this.isSingleActiveTriggerNode && !this.isTriggerNodeTooltipEmpty && !this.isNodeDisabled && !this.hasIssues) {
+					this.showWebhookNodeTooltip = !isDragActiveTriggerNode;
+				}
+			},
+		},
+		nodePosition: {
```

Comment:
as discussed might not be needed because of drag

Reply:
Done